inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/util/StringUtil.java

```diff
@@ -106,4 +245,26 @@ public static String formatDate(Date date) {
         return sdf.format(date);
     }
 
+    public static long parseDateTime(String value) {
+        try {
+            if (value.length() < 8) {
```

Comment:
What do the 8 and 9 mean here?
Suggested adding a comment for them.

Reply:
Add the reason in the java doc, thx.